catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -222,7 +222,7 @@
             "Unable to retrieve product for ID: %s";
 
     private static final List<String> TYPE_NAMES_LIST = Arrays.asList(CswConstants.CSW_RECORD,
-            CswConstants.GMD_METADATA_TYPE);
+            GmdMetacardType.GMD_METACARD_TYPE_NAME);
```

Comment:
Nit. It's a little weird we use the Metacard type for one constant and then use CswConstants for the other